notebooks/official/pipelines/lightweight_functions_component_io_kfp.ipynb

```diff
@@ -1,874 +1,771 @@
+    "    pipeline_root=PIPELINE_ROOT,\n",
+    "    # A name for the pipeline. Use to determine the pipeline Context.\n",
+    "    name=\"metadata-pipeline-v2\",\n",
+    ")\n",
+    "def pipeline(message: str):\n",
+    "    importer = kfp.dsl.importer(\n",
+    "        artifact_uri=\"gs://ml-pipeline-playground/shakespeare1.txt\",\n",
+    "        artifact_class=Dataset,\n",
+    "        reimport=False,\n",
+    "    )\n",
+    "    preprocess_task = preprocess(message=message)\n",
+    "    train_task = train(\n",
+    "        dataset_one_path=preprocess_task.outputs[\"output_dataset_one\"],\n",
+    "        dataset_two=preprocess_task.outputs[\"output_dataset_two_path\"],\n",
+    "        imported_dataset=importer.output,\n",
+    "        message=preprocess_task.outputs[\"output_parameter_path\"],\n",
+    "        num_steps=5,\n",
+    "    )\n",
+    "    read_task = read_artifact_input(  # noqa: F841\n",
+    "        generic=train_task.outputs[\"generic_artifact\"]\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "compile_pipeline"
+   },
+   "source": [
+    "## Compile the pipeline\n",
+    "\n",
+    "Next, compile the pipeline."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "compile_pipeline"
+   },
+   "outputs": [],
+   "source": [
+    "compiler.Compiler().compile(\n",
+    "    pipeline_func=pipeline, package_path=\"lightweight_pipeline.yaml\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "run_pipeline:lightweight"
+   },
+   "source": [
+    "## Run the pipeline\n",
+    "\n",
+    "Next, run the pipeline."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "run_pipeline:lightweight"
+   },
+   "outputs": [],
+   "source": [
+    "DISPLAY_NAME = \"shakespeare\"\n",
+    "\n",
+    "job = aiplatform.PipelineJob(\n",
+    "    display_name=DISPLAY_NAME,\n",
+    "    template_path=\"lightweight_pipeline.yaml\",\n",
+    "    pipeline_root=PIPELINE_ROOT,\n",
+    "    parameter_values={\"message\": \"Hello, World\"},\n",
+    ")\n",
+    "\n",
+    "job.run()\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "view_pipeline_run:lightweight"
+   },
+   "source": [
+    "Click on the generated link to see your run in the Cloud Console.\n",
+    "\n",
+    "<!-- It should look something like this as it is running:\n",
+    "\n",
+    "<a href=\"https://storage.googleapis.com/amy-jo/images/mp/automl_tabular_classif.png\" target=\"_blank\"><img src=\"https://storage.googleapis.com/amy-jo/images/mp/automl_tabular_classif.png\" width=\"40%\"/></a> -->\n",
+    "\n",
+    "In the UI, many of the pipeline DAG nodes will expand or collapse when you click on them. Here is a partially-expanded view of the DAG (click image to see larger version).\n",
```

Comment:
- Question, is "UI reference to the Google Cloud console? If so, change "UI" to "Google Cloud console.

- remove "will" from "will expand"